backend/app/Http/Controllers/Api/PageTimingController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\System\FilterByPeriodHttpRequest;
+use App\Http\Resources\ButtonResource;
+use App\Http\Response\ApiResponse;
+use App\Actions\PageTimings\{GetAverageTimingAction, GetAverageTimingRequest};
+
+final class PageTimingController extends Controller
+{
+    private $getAveragePageLoadTimeAction;
+
+    public function __construct(
+        GetAverageTimingAction $getAveragePageLoadTimeAction
+    ) {
+        $this->getAveragePageLoadTimeAction = $getAveragePageLoadTimeAction;
+    }
+
+    public function getAveragePageLoading(FilterByPeriodHttpRequest $request): ApiResponse
+    {
+        $average = $this->getAveragePageLoadTimeAction->execute(
+            GetAverageTimingRequest::fromRequest($request),
+            'page_load_time'
```

Comment:
this prameter should be inside request. You can put request to variable and set it after all.

$averageRequest  = GetAverageTimingRequest::fromRequest($request);
$averageRequest->setParameter('page_load_time');

or event better to create from constructor

```
$averageRequest  = new GetAverageTimingRequest(
   $request->getStartDate(),
   $request->getEndDate(),
   'page_load_time'
);
```